components/app/explore/similar-datasets/similar-datasets-component.js

```diff
@@ -0,0 +1,46 @@
+import React, { PureComponent } from 'react';
+import PropTypes from 'prop-types';
+
+// Redux
+import { connect } from 'react-redux';
+
+// Components
+import DatasetList from 'components/app/explore/DatasetList';
+import Spinner from 'components/ui/Spinner';
+
+class SimilarDatasets extends PureComponent {
+  render() {
+    const { similarDatasets } = this.props;
+    const { loading, data, error } = similarDatasets;
+
+    return (
+      <div className="c-similar-datasets">
+        <Spinner isLoading={loading} className="-light" />
+        <h3>Similar datasets</h3>
+        {data && data.length > 0 &&
+          <DatasetList
+            active={[]}
```

Comment:
Should `DatasetList`have a default prop active set as an empty array instead of passing it empty?

Reply:
yeah, I agree 👍